checkov/cloudformation/runner.py

```diff
@@ -40,7 +40,10 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
 
             for file in files_list:
                 relative_file_path = f'/{os.path.relpath(file, os.path.commonprefix((root_folder, file)))}'
-                (definitions[relative_file_path], definitions_raw[relative_file_path]) = parse(file)
+                try:
+                    (definitions[relative_file_path], definitions_raw[relative_file_path]) = parse(file)
+                except TypeError:
+                    logging.info(f'CloudFormation skipping {file} as it is not a valid CF template')
```

Comment:
@schosterbarak Handles cases where parsing of CF file failed. there are many `yaml` and `json` files, and some although have blocks similar to CF templates, are not CF templates. we are using the AWS [cfn_yaml](checkov/cloudformation/parser/cfn_yaml.py) parser. So if it fails - it's safe to assume this is not a valid CFN template